### ISSUE TITLE
Plugin Crash on 0.3.7

### ISSUE THREAD

**Issue by indietyp** [koxudaxi/pydantic-pycharm-plugin#372]
**Describe the bug**
pydantic crashes.

**To Reproduce**
1. Open PyCharm

**Expected behavior**
The plugin does not crash

**Screenshots**
Also, gif movies are recommended.

**Environments (please complete the following information):**
 - IDE: PyCharm Professional 2021.2.2
 - OS: macOS 11.5.2
 - Pydantic Version 1.8.2
 - Plugin version 0.3.7

**Additional context**

```
java.lang.NoSuchMethodError: 'com.jetbrains.python.psi.types.PyType com.jetbrains.python.psi.types.PyLiteralType$Companion.promoteToLiteral(com.jetbrains.python.psi.PyExpression, com.jetbrains.python.psi.types.PyType, com.jetbrains.python.psi.types.TypeEvalContext)'
	at com.koxudaxi.pydantic.PydanticTypeCheckerInspection$Visitor.analyzeCallee(PydanticTypeCheckerInspection.kt:112)
	at com.koxudaxi.pydantic.PydanticTypeCheckerInspection$Visitor.checkCallSiteForPydantic(PydanticTypeCheckerInspection.kt:52)
	at com.koxudaxi.pydantic.PydanticTypeCheckerInspection$Visitor.visitPyCallExpression(PydanticTypeCheckerInspection.kt:42)
	at com.jetbrains.python.psi.impl.PyCallExpressionImpl.acceptPyVisitor(PyCallExpressionImpl.java:27)
	at com.jetbrains.python.psi.impl.PyBaseElementImpl.accept(PyBaseElementImpl.java:69)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:64)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$7(LocalInspectionsPass.java:353)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:172)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:130)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:119)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:109)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$10(LocalInspectionsPass.java:353)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:136)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1078)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:92)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:104)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:83)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:183)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:83)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

sorry for so many bug reports^^.

**Comment by koxudaxi** [comment]
@indietyp

I'm sorry!! I thought the changes from `2021.3`
https://github.com/koxudaxi/pydantic-pycharm-plugin/pull/350

I just pushed a new release to the market. We must wait for approval 

> sorry for so many bug reports^^.

It's very important for us. Because I can't test all my projects every time.
Thank you very much 🙇

**Comment by koxudaxi** [comment]
@indietyp @tonal @jvlar
Fixed version 0.3.8 is just released !!
